run.mac.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+osname=$(uname)
```

Comment:
yes, using the aws command line tools.

One more thing, you can run cli and browser mode at the same time, if you have made two git clones, one running cli and the other browser mode (I have not checked this out -- but celery instances for each would be separate)